### PR TITLE
chore: Print SSO errors to console

### PR DESCRIPTION
**Pull request by Yserz**
Related: https://github.com/wireapp/wire-webapp/pull/6480

### CODE REVIEW

File: src/script/auth/page/SingleSignOnForm.tsx [hunk 1]
```diff
@@ -187,11 +187,11 @@ class _SingleSignOnForm extends React.PureComponent<Props & ConnectedProps & Dis
...
           default: {
             this.setState({ssoError: error});
-            const isValidationError = Object.values(ValidationError.ERROR).some(errorType =>
-              error.label.endsWith(errorType)
+            const isValidationError = Object.values(ValidationError.ERROR).some(
+              errorType => error.label && error.label.endsWith(errorType)
             );
             if (!isValidationError) {
-              throw error;
+              console.warn('SSO authentication error', JSON.stringify(Object.entries(error)));
```
Comment (ffflorian) [inline src/script/auth/page/SingleSignOnForm.tsx]:
```suggestion
              console.warn('SSO authentication error', JSON.stringify(Object.entries(error)), error);
```

Comment (Yserz) [reply src/script/auth/page/SingleSignOnForm.tsx]: There's no value in doing that because the error doesn't contain any useful data in this form.

Comment (Yserz) [reply src/script/auth/page/SingleSignOnForm.tsx]: image

Comment (ffflorian) [reply src/script/auth/page/SingleSignOnForm.tsx]: But you will get the stack trace.

File: src/script/auth/page/SingleSignOnForm.tsx [hunk 2]
```diff
@@ -187,11 +187,11 @@ class _SingleSignOnForm extends React.PureComponent<Props & ConnectedProps & Dis
...
           default: {
             this.setState({ssoError: error});
-            const isValidationError = Object.values(ValidationError.ERROR).some(errorType =>
-              error.label.endsWith(errorType)
+            const isValidationError = Object.values(ValidationError.ERROR).some(
+              errorType => error.label && error.label.endsWith(errorType)
             );
             if (!isValidationError) {
-              throw error;
+              console.warn('SSO authentication error', JSON.stringify(Object.entries(error)), error);
```
Comment (fisx) [inline src/script/auth/page/SingleSignOnForm.tsx:194]: would it make sense to throw the error as before, but log the error unconditionally?

```
             console.info('SSO authentication error', JSON.stringify(Object.entries(error)), error);
             if (!isValidationError) {
                 throw error;
             }
```

no strong opinion there, i'd be happy with your change as is (given it works as discussed :).  and i haven't tested the `console.info` thing.

**Comment by fisx** [comment]
thanks!

File: src/script/auth/page/SingleSignOnForm.tsx (hunk 2 above)
Comment (Yserz) [reply src/script/auth/page/SingleSignOnForm.tsx:194]: Basically we filter for known errors not related to SSO. In other words all SSO errors are logged here.

**Pull request merged**